src/main/resources/application.conf

```diff
@@ -12,6 +12,7 @@ conseildb = {
     databaseName = "conseil"
     user = "foo"
     password = "bar"
+    reWriteBatchedInserts = true
```

Comment:
Is this used anywhere?

Reply:
it's the configuration that actually makes the underlying postgres jdbc driver rewrite multiple single inserts to one batch insert...

Reply:
actually this is the only relevant change for the PR, the other are just refactors with no real effect